### ISSUE TITLE
multiple numbers in a digit

### ISSUE THREAD

**Issue by TifeXYZ** [byZeroOfficial/Counter#4]

**Comment by TifeXYZ** [comment]
![Screenshot_20220215-220341.png](https://user-images.githubusercontent.com/76390194/154148848-f85e1b59-a5fb-485d-9201-7a56588fd2a9.png)
![Screenshot_20220215-220336.png](https://user-images.githubusercontent.com/76390194/154148859-5cb63ecd-3088-415b-b9c0-9f0a929e32f7.png)

**Comment by byZeroOfficial** [comment]
Yeah, that won't work. You need to put every number in a separate field
